### PR TITLE
Bugfix/ci

### PR DESCRIPTION
**Pull request by Roshan-y**

### CODE REVIEW

File: .github/workflows/sonarcloud.yml [hunk 1]
```diff
@@ -40,74 +46,59 @@ jobs:
           distribution: "temurin"
           java-version: "17"
 
-      # Caching is a very useful part of a CI, as a workflow is executed in a clean environment every time,
+      # Caching is a very useful part of a CI, as a workflow is executed in a clean environement every time,
```
Comment (robertaleG) [inline .github/workflows/sonarcloud.yml:49]: _environment_ is spelled _environment_ not _environement_

**Review by robertaleG — CHANGES_REQUESTED** [review]
The code works and solves the CI issue. There is still an issue with sonar cloud. I recommend you try to add this line in _AndroidManifest.xml_ :
```
<application
    android:usesCleartextTraffic="false"   // add this line
    ... >
    ...
</application>
```
This should neutralise the warning.

Also, please correct the spelling mistake on line 49. 
After making those changes and if they pass, we can merge your PR.

File: .github/workflows/sonarcloud.yml [hunk 2]
```diff
@@ -40,74 +44,54 @@ jobs:
           distribution: "temurin"
           java-version: "17"
 
-      # Caching is a very useful part of a CI, as a workflow is executed in a clean environment every time,
-      # this means that one would need to re-download and re-process gradle files for every run. Which is very time consuming.
-      #
       # To avoid that, we cache the the gradle folder to reuse it later.
```
Comment (aaisel) [inline .github/workflows/sonarcloud.yml:47]: Remove comment

**Comment by aaisel** [comment]
Merging this to enable CI

**Review by aaisel — APPROVED** [review]

**Pull request merged**